Dockerfile.processing_base

```diff
@@ -23,5 +23,6 @@ RUN R -e "BiocManager::install('SingleCellExperiment')"
 #       See https://github.com/cellgeni/sceasy#installation
 RUN R -e "BiocManager::install('LoomExperiment')"
 RUN R -e "install.packages(c('devtools', 'igraph'));"
-RUN R -e "library(devtools); devtools::install_github('cellgeni/sceasy')"
+# Pinning sceasy version via commit hash since tags aren't maintained in that Github repo
+RUN R -e "library(devtools); devtools::install_github('cellgeni/sceasy', ref='57cada7')"
```

Comment:
`57cada7` is the commit hash which includes the recent uns metadata fix:
-  https://github.com/cellgeni/sceasy/commit/57cada717e0abd8bf77920d53dd30dc05765aff1